sdk/ai/azure-ai-generative/sdk_packaging.toml

```diff
@@ -0,0 +1,11 @@
+[packaging]
+package_name = "azure-ai-generative"
+package_nspkg = "azure-ai-nspkg"
+package_pprint_name = "MyService Management"
```

Comment:
Not sure if this is typo. (shouldn't it be something like "Azure AI Generative"?

Reply:
It's because a new CI bot which generates the information wrong. It's disabled in the next revision